synchronization/src/main/java/de/cyface/synchronization/WiFiSurveyor.java

```diff
@@ -60,11 +64,6 @@
      * The current Android context (i.e. Activity or Service).
      */
     private WeakReference<Context> context;
-    /**
-     * A flag that might be queried to see whether synchronization is active or not. This is <code>true</code> if
-     * synchronization is active and <code>false</code> otherwise.
-     */
-    private boolean synchronizationIsActive;
```

Comment:
This attribute was misleading. As long as we do not directly cancel synchronization when we change the auto-sync settings this would say there is no active sync while there may be still an ongoing active sync (e.g. serialization).
There is a new task to add cancelation but as I see it it's not so urgent right now.